src/post_iterate.rs

```diff
@@ -0,0 +1,23 @@
+use crate::fabric::{Fabric, UniqueId};
+
+// how to do this?
+pub trait PostIterate {
```

Comment:
Should this maybe be done with a rich enum with different PostIterate implementations or something?

Reply:
No, i think you're doing it right! This is a good case to use `&dyn PostIterate` or `&[&dyn PostIterate]`. Will you have multiple PostIterate operations?